docs/book/src/plugins/available/autoupdate-v1-alpha.md

```diff
@@ -11,10 +11,12 @@ refresh your project scaffold, and wraps it in a GitHub Actions workflow that op
 
 - When you don’t deviate too much from the default scaffold — ensure that you see the note about customization [here](https://book.kubebuilder.io/versions_compatibility_supportability#project-customizations).
 - When you want to reduce the burden of keeping the project updated and well-maintained.
+- When you want to guidance and help from AI to know what changes are needed to keep your project up to date
+as to solve conflicts.
 
 ## How to Use It
 
-0 If you want to add the `autoupdate` plugin to your project:
+- If you want to add the `autoupdate` plugin to your project:
 
 ```shell
 kubebuilder edit --plugins="autoupdate.kubebuilder.io/v1-alpha"
```

Comment:
<img width="334" height="624" alt="Screenshot 2025-08-24 at 02 29 36" src="https://github.com/user-attachments/assets/3568ff77-7d7d-4aa8-950a-49ace627ec07" />

Reply:
@vitorfloriano that is the final result :-)